app/src/main/java/org/sopt/sample/week1/login/LoginActivity.kt

```diff
@@ -0,0 +1,69 @@
+package org.sopt.sample.week1.login
+
+import android.content.Intent
+import android.os.Bundle
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import org.sopt.sample.databinding.ActivityLoginBinding
+import org.sopt.sample.week1.common.Constants
+import org.sopt.sample.week1.common.MBTI
+import org.sopt.sample.week1.mypage.MyPageActivity
+import org.sopt.sample.week1.signup.SignUpActivity
+
+class LoginActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityLoginBinding
+    private lateinit var insertedId: String
+    private lateinit var insertedPassword: String
+    private lateinit var insertedMbti: MBTI
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        binding = ActivityLoginBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        // intent 에 값이 로그인 정보가 존재하는 경우
+        if (intent.hasExtra(Constants.ID.value) && intent.hasExtra(Constants.PASSWORD.value)) {
+            insertedId = intent.getStringExtra(Constants.ID.value)
+                ?: throw java.lang.IllegalArgumentException("id expected")
```

Comment:
```suggestion
                ?: throw IllegalArgumentException("id expected ${intent.getStringExtra(Constants.ID.value)}")
```

- Kotlin exception을 활용해주세요
- Exception이 터질 경우 해당 에러가 발생한 원인이 되는 값을 찍어주는 것이 좋습니다.